en/micro-integrator/docs/setup/deployment/k8s-pipeline/modifying-pipeline-configurations.md

```diff
@@ -0,0 +1,201 @@
+# Modifying Pipeline Configurations
+
+## The pipeline configurations can be changed by the modification of values.yaml file
+
+The pipeline configurations can be changed by the modification of the
+configurations in
+the [values.yaml](https://github.com/wso2/kubernetes-pipeline/blob/master/kubernetes-pipeline/values.yaml) file
+corresponding to the pipeline. The [sample
+modifications](https://github.com/wso2/kubernetes-pipeline/tree/master/kubernetes-pipeline/samples)
+for each product provide a good indication as to how these
+configurations could be modified upon deployment. Once a configuration
+change has been made, run the helm upgrade command to apply the changes.
+
+``` bash
+$ helm upgrade <RELEASE_NAME> wso2/kubernetes-pipeline --version 1.0.0 -f values.yaml
+```
+>
+
+**<RELEASE\_NAME\>** should be replaced with the release name provided
+when the pipeline is installed initially.
+
+Configuration changes occur which will affect the Jenkins
+deployment, some of which include:
+
+  * Changes to the private Docker repository
+
+  * Changes to the artifact source repository
+
+  * Changes to the chart source repository
+
+In these cases, the **Jenkins pod would be restarted for these
+configurations to be reflected**.
+
+Following are some advanced configurations that could be done to
+customize and secure the pipeline tools.
+
+## Change Domain
+
+To customize the domain name override the host values as shown in the
+example below.
+
+``` yml
```

Comment:
Does this render properly on the site?

Reply:
![image](https://user-images.githubusercontent.com/40120734/90508423-5dd6c180-e175-11ea-9345-540d7f7e52d3.png)
this render properly